internal/service/networkfirewall/firewall_policy.go

```diff
@@ -327,6 +331,11 @@ func expandStatefulRuleGroupReferences(l []interface{}) []*networkfirewall.State
 		if v, ok := tfMap["resource_arn"].(string); ok && v != "" {
 			reference.ResourceArn = aws.String(v)
 		}
+		if v, ok := tfMap["override_action"].(string); ok && v != "" {
+			override := &networkfirewall.StatefulRuleGroupOverride{}
+			override.SetAction(v)
+			reference.Override = override
+		}
```

Comment:
Since we are using `action` nested in `override`, we will use another `expand` function in order to extract the `action` argument.

Reply:
Updated.